assets/out

```diff
@@ -63,7 +63,7 @@ for i in $(seq 0 $(expr "$files_count" - 1)); do
 done
 
 if [ ! -z "$build" ]; then
-  osc build $repository $arch
+  osc build --trust-all-projects $repository $arch
```

Comment:
Can this also be set in the .oscrc file? Since we already [create one](https://github.com/SUSE/open-build-service-resource/blob/master/assets/common.sh#L10), maybe it is a good idea to keep all options there for quicker overview of what is being used.